hathor/nanocontracts/nc_types/__init__.py

```diff
@@ -118,7 +119,7 @@
     Amount: VarUint32NCType,
     BlueprintId: Bytes32NCType,
     ContractId: Bytes32NCType,
-    Timestamp: Int32NCType,
```

Comment:
Is this change compatible with the current implementation in the clients? If not, we should add it to the next release notes.

Reply:
The only compatibility caveat is that a timestamp higher than `2**31 - 1` (which is `2038-01-19T04:14:07` in ISO format), would previously be deserialized to negative values, which could cause unexpected behavior.

I haven't checked if it affects any actual transaction in the current network, but I think it's highly unlikely that it does.

Clients shouldn't be affected, unless they incorrectly use negative timestamp values, but I'll align this with Pedro anyway.